_sass/layout/_header.scss

```diff
@@ -11,6 +11,10 @@
   background-color: _palette(accent1);
   cursor: default;
   padding: 1.75em 2em;
+  position: -webkit-sticky;
```

Comment:
Is this still needed anymore? Appears to work without prefix according to [canise.com](https://caniuse.com/#feat=css-sticky)

Reply:
Yes and no.  We don't have `postcss` installed so I used `autoprefixer`'s [website](https://autoprefixer.github.io/) to target the [last 4 versions](https://browserl.ist/?q=last%204%20version) of each major browser:

<img width="913" alt="image" src="https://user-images.githubusercontent.com/6334517/79076699-77449a00-7cb9-11ea-9851-a1d915a4e756.png">

Reply:
> Yes and no.  We don't have `postcss` installed so I used `autoprefixer`'s [website](https://autoprefixer.github.io/) to target the [last 4 versions](https://browserl.ist/?q=last%204%20version) of each major browser:
> 
> <img width="913" alt="image" src="https://user-images.githubusercontent.com/6334517/79076699-77449a00-7cb9-11ea-9851-a1d915a4e756.png">
> 

Ok. Seems reasonable!